include/RAJA/config.hpp.in

```diff
@@ -218,16 +218,16 @@ static_assert(RAJA_HAS_SOME_CXX14,
  *
  ******************************************************************************
  */
-#if defined(__INTEL_COMPILER)
+#if defined(_WIN32)
```

Comment:
If it always must be defined for windows, should it be in it's own conditional logic rather than part of the if-elif-.... logic?

Reply:
@adayton1 what are your thoughts on my question above? Should we merge this?

Reply:
Makes sense. Please add an issue to possibly rework at some point. Thank you.

Reply:
Added #1466